### ISSUE TITLE
WCAG 2.2 Violation - There are controls present within the interface with visible text that is not included as part of the controls' accessible names.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#12]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

#### Further information:

1. 

### Instances

- #36
